### ISSUE TITLE
NFTs moved to Private mode

### ISSUE THREAD

**Issue by SAClough** [zeromintapp/community#14]
I have an artist on my site Artboom and we tried selling ten copies of his NFTs for $500 each.
Then we decided to try having just one but at $5000 each.
The previous NFTs can't be cancelled or destroyed so I set up a new Zeromint account for him, Robert Lally, and put the old account as Private Collection thinking to just close it somehow to 'cap it off' so to speak.
Problem is Google keeps displaying not only the page showing ten copies for sale at $500 but the Gochain explorer contract for it.
I wondered if it could be possible to add a 'do not follow' tag to things when they are moved to Private so that Google will eventually stop displaying them in its search results. #

**Comment by gobullsclub** [comment]
Good Idea and Point @SAClough ! Had the problem as well...according to @r-gochain @treeder as Owner of the NFTs it will always show on the User NFTs section (Its Private) Nobody Could see it. People who have the Links to the NFTs will able to view as Page Not Found.
![NotFoundError](https://user-images.githubusercontent.com/97951436/150667502-9ed0685f-9547-44a7-a5e1-855a27632b2c.png)
Its better to say " This Collection is Private ,You can't View or Buy This NFTs" 
What happens if the Owner of a NFT collection decided to make the Collection private with other Users holding the NFTs ?
So it is Wise to have a Delete Collection (With Pending Approval by Zeromint Team) Rather than Making the Collection Private
